run.sh

```diff
@@ -0,0 +1,52 @@
+
+#!/bin/sh
+#
+# Run Ylitse app on a device.
+
+#set the API_URL variable
+if [[ ${YLITSE_API_URL} && ${YLITSE_LOGIN_URL} && ${YLITSE_TERMS_URL} && ${YLITSE_USERGUIDE_URL} && ${YLITSE_FEEDBACK_URL} ]]; then
+    API_URL=${YLITSE_API_URL}
+    LOGIN_URL=${YLITSE_LOGIN_URL}
+    TERMS_URL=${YLITSE_TERMS_URL}
+    FEEDBACK_URL=${YLITSE_FEEDBACK_URL}
+    USERGUIDE_URL=${YLITSE_USERGUIDE_URL}
+else
+    API_URL="localhost"
+    LOGIN_URL="localhost"
+    TERMS_URL="localhost"
+    FEEDBACK_URL="localhost"
+    USERGUIDE_URL="localhost"
+fi
+
+set -eu
+
+print_help() {
+    echo "Usage: ./run {android|ios}"
+}
+
+FILE=config.json
+if test -f "$FILE"; then
+    echo "$FILE exists"
```

Comment:
I am bit sceptical on this one as default behaviour.

If I want to first run against localhost, then against dev server and after that I want to release a version, I see a risk of not removing file before setting things upp. But we can go with this one and change it if it feels clumsy.